### ISSUE TITLE
Changing hostname indemnification to L0 keeps DNSSEC findings

### ISSUE THREAD

**Issue by zcrt** [minvws/nl-kat-coordination#1080]
**Describe the bug**
Changing a hostname indemnification to L0 keeps DNSSEC findings while removing DKIM, DMARC and SPF findings.

**To Reproduce**
1. Scan hostname with no DNSSEC and no DKIM/DMARC/SPF
2. Make sure the findings are there
3. Change hostname indemnification to L0
4. Notice the DKIM/DMARC/SPF findings disappear, but the DNSSEC finding stays.

**Expected behavior**
Removal of DNSSEC finding

**OpenKAT version**
1.8.0.1

**Comment by noamblitz** [comment]
DNSSEC findings are created by a boefje and not by a bit. Therefore this is not a bug. Since the DNSSEC boefje will not rerun when clearance is set to L0, the findings will not disappear.

**Comment by zcrt** [comment]
Sounds to me that the bug is then either:
- Findings are directly created by a boefje instead of leaving that to a bit
- Findings are not disappearing when their original input (e.g. hostname) is no longer a valid input (`L0`) for the boefje(s) that created the finding

**Comment by noamblitz** [comment]
Well thats not really a bug imo. 

Prohibiting boefjes from creating findings, in this case, can be a choice indeed. But thats really more a choice than a bug.
Also, removing clearence - from now on - means that that ooi cannot be scanned anymore, removing all data gathered by boefjes about that ooi should not happen imo.

**Comment by zcrt** [comment]
Agreed that gathered data should not be removed. However, in my opinion it should end up as historical data. E.g. if you search for the findings at a date where the hostname still had L1 the finding should be visible. If you search in the most recent view while the hostname has L0, they should not exist however.

**Comment by noamblitz** [comment]
Hmm that could be a nice feature indeed. Quite difficult to follow the trail and set all concurrent oois as historical as well, but i like the idea!

**Comment by noamblitz** [comment]
For sure a good idea, but indeed, showing affected objects is hard
